ci/ci/build.py

```diff
@@ -585,6 +593,9 @@ def build(self, batch, code, scope):  # pylint: disable=unused-argument
 date
 '''
 
+        if is_test_deployment:
+            script = 'true'
```

Comment:
Isn't it easier to just skip the job an return at the top?

Reply:
I didn't know if we had to return a job if there's dependencies on that step.

Reply:
We don't return a job, we assign to the job attribute of steps.  By default it is None.  When computing dependencies, we add dependencies to all non-None jobs of parent steps.  So, in a word, nope!